### ISSUE TITLE
--debug leads to logging error after composing new entry

### ISSUE THREAD

**Issue by micahellison** [jrnl-org/jrnl#1496]
### Environment
  - `jrnl --diagnostic` output: 
```
jrnl: v2.8.4
Python: 3.9.7 (tags/v3.9.7:1016ef3, Aug 30 2021, 20:19:38) [MSC v.1929 64 bit (AMD64)]
OS: Windows 10
```
  - Install method:  pipx

### Current Behavior

Something is causing the debug logger to crash in write mode. It dumps a traceback. Originally reported in #1494 but this is a separate issue than that issue's primary issue.

### Expected Behavior

The logger shouldn't crash on normal inputs.

### Repro Steps

```
jrnl --debug this is a new entry
```

### Other Information

Stack trace happens right after `[Entry added to default journal]`:
```
--- Logging error ---
Traceback (most recent call last):
  File "C:\Users\micah\.pyenv\pyenv-win\versions\3.9.7\lib\logging\__init__.py", line 1083, in emit
    msg = self.format(record)
  File "C:\Users\micah\.pyenv\pyenv-win\versions\3.9.7\lib\logging\__init__.py", line 927, in format
    return fmt.format(record)
  File "C:\Users\micah\.pyenv\pyenv-win\versions\3.9.7\lib\logging\__init__.py", line 663, in format
    record.message = record.getMessage()
  File "C:\Users\micah\.pyenv\pyenv-win\versions\3.9.7\lib\logging\__init__.py", line 367, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "C:\Users\micah\.pyenv\pyenv-win\versions\3.9.7\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\micah\.pyenv\pyenv-win\versions\3.9.7\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\micah\.local\bin\jrnl.exe\__main__.py", line 7, in <module>
    sys.exit(cli())
  File "C:\Users\micah\.local\pipx\venvs\jrnl\lib\site-packages\jrnl\cli.py", line 34, in cli
    return run(args)
  File "C:\Users\micah\.local\pipx\venvs\jrnl\lib\site-packages\jrnl\jrnl.py", line 69, in run
    write_mode(**kwargs)
  File "C:\Users\micah\.local\pipx\venvs\jrnl\lib\site-packages\jrnl\jrnl.py", line 149, in write_mode
    logging.debug("Write mode: completed journal.write()", args.journal_name, raw)
Message: 'Write mode: completed journal.write()'
Arguments: ('default', 'this is a new entry')
```

**Comment by apainintheneck** [comment]
I'm not able to reproduce this error on macOS.

**Comment by jonakeys** [comment]
This is already fixed with PR #1484

**Comment by micahellison** [comment]
> This is already fixed with PR #1484

Thanks @jonakeys for the heads up! I just double-checked and confirmed that it works on the develop branch.
